api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/x509/X509Properties.java

```diff
@@ -245,6 +245,12 @@
      */
     private String sslHeaderName = DEFAULT_CERT_HEADER_NAME;
 
+    /**
+     * Attribute name that may be used by some X509 principal resolvers if the main attribute in the
```

Comment:
The note here is sort of problematic: if the property belongs in `X509Properties`, it means that it should work for all X509 related functionality/resolvers. Saying it works for *some* and not others immediately begs the question of: "which ones?". So in principle, the setting should work with everything relevant, or it should be moved to properties class that actually can do something meaningful with the setting.